docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,34 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Correlations editor in Explore
+
+<!-- Kristina Durivage -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem -->
+
+<Date>
```

Comment:
Looking for an approximate release date here

Reply:
It was made available for fast channel on 2023/10/03.